docs/source/conf.py

```diff
@@ -347,6 +347,7 @@ def inject_weight_metadata(app, what, name, obj, options, lines):
             metrics = meta.pop("_metrics")
             for dataset, dataset_metrics in metrics.items():
                 for metric_name, metric_value in dataset_metrics.items():
+                    metric_name = metric_name.replace("_", "-")
```

Comment:
Should we do this? It's the documentation that shows the actual name of the variable. (I know you made it private but still)